context/rootfs/scripts/init-registry.sh

```diff
@@ -67,20 +90,21 @@ regArgs="-d --restart=always \
 -e REGISTRY_HTTP_TLS_KEY=/certs/$REGISTRY_DOMAIN.key"
 
 if [ -f "$config" ]; then
-  sed -i "s/5000/$1/g" "$config"
-  regArgs="$regArgs \
+    sed -i "s/5000/${REGISTRY_PORT}/g" "$config"
+    regArgs="$regArgs \
     -v $config:/etc/docker/registry/config.yml"
 fi
 
 if [ -f "$htpasswd" ]; then
-  nerdctl run "$regArgs" \
-    -v "$htpasswd":/htpasswd \
-    -e REGISTRY_AUTH=htpasswd \
-    -e REGISTRY_AUTH_HTPASSWD_PATH=/htpasswd \
-    -e REGISTRY_AUTH_HTPASSWD_REALM="Registry Realm" registry:2.7.1 || startRegistry
+    $imageServer run "$regArgs" \
+            -v "$htpasswd":/htpasswd \
+            -e REGISTRY_AUTH=htpasswd \
+            -e REGISTRY_AUTH_HTPASSWD_PATH=/htpasswd \
+            -e REGISTRY_AUTH_HTPASSWD_REALM="Registry Realm" registry:2.7.1 || start_registry
 else
-  nerdctl run "$regArgs" registry:2.7.1 || startRegistry
+    $imageServer run "$regArgs" registry:2.7.1 || start_registry
```

Comment:
remove ""

Reply:
[SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.